types/ramda/test/nth-tests.ts

```diff
@@ -2,8 +2,15 @@ import * as R from 'ramda';
 
 () => {
   const list = ['foo', 'bar', 'baz', 'quux'];
+  const str = 'abcd';
+
   R.nth(1, list); // => 'bar'
   R.nth(-1, list); // => 'quux'
   R.nth(-99, list); // => undefined
   R.nth(-99)(list); // => undefined
+  R.nth(1, str); // => 'b'
```

Comment:
Can you add `$ExpectType` to do type tests?

Use $ExpectType to assert that an expression is of a given type, and $ExpectError to assert that a compile error.

Reply:
I agree with this sentiment. You don't need them for all of the tests you've written, but one that demonstrates each overload would be nice.

Reply:
I modified the tests and added $ExpectType to each one